src/sirhelpers.c

```diff
@@ -323,16 +323,15 @@ struct tm* _sir_localtime(const time_t* restrict timer, struct tm* restrict buf)
         }
 
         return buf;
-# else /* __WIN__ */
-        struct tm* ret = localtime_s(timer, buf);
+# else /* !__WIN__ */
```

Comment:
Could you flip it around? We always do if !__WIN__ first because that's the primary target...